src/status_im/chat/models/loading.cljs

```diff
@@ -124,15 +113,28 @@
           (reduce (fn [{:keys [all-messages] :as acc}
                        {:keys [message-id from]
                         :as   message}]
-                    (cond-> acc
-                      (not (get-in db [:chats chat-id :users from]))
-                      (update :senders assoc from message)
-
-                      (nil? (get all-messages message-id))
-                      (update :new-messages conj message)
-
-                      :always
-                      (update :all-messages assoc message-id message)))
+                    (let [message
+                          ;; For example, when a user receives a list of 4 image messages while inside
+                          ;; the chat screen we
+                          ;; shouldn't group the images into albums. When the user exists the chat screen
+                          ;; then enters the
+                          ;; chat screen again, we now need to group the images into albums (like
+                          ;; WhatsApp). The albumize?
+                          ;; boolean is used to know whether we need to group these images into albums
+                          ;; now or not. The
+                          ;; album-id can't be used for this because it will always be there.
+                          (if (get all-messages message-id)
+                            message
+                            (assoc message :albumize? true))]
```

Comment:
are we marking all messages with `albumize`?

Reply:
So a new audio message will be marked?

Reply:
Oh I see. Will fix soon.

Reply:
Done